src/css/_heatmap.scss

```diff
@@ -0,0 +1,9 @@
+@mixin heatmap($theme-name, $theme-colors) {
+    .heatmap {
+      width: 100%;
+      left: 0;
+      position: absolute;
+      padding-right: inherit;
+      padding-left: inherit;
+    }
+}
```

Comment:
Your explanation on the PR is great! Could you paste it as a comment, perhaps here? (Not to be super-paranoid, but the PR comments are locked into github, but comments in the code will always be here.)